UrologicalSupplies/R4/resources/Questionnaire-R4-UrologicalSupplies.json

```diff
@@ -0,0 +1,352 @@
+        }
+      ]
+    },
+    {
+      "linkId": "3",
+      "type": "group",
+      "required": false,
+      "text": "Order Start Date",
+      "item": [
+        {
+          "type": "date",
+          "required": false,
+          "linkId": "3.1",
+          "text": "Order start date, if different from date of order"
+        }
+      ]
+    },
+    {
+      "linkId": "4",
+      "required": false,
+      "text": "Order Type",
+      "type": "open-choice",
+      "item": [
+        {
+          "linkId": "4.1",
+          "text": "Type of Order",
+          "type": "open-choice",
+          "answerValueSet": "http://hl7.org/fhir/ValueSet/request-intent"
+        }
+      ]
+    },
+    {
+      "linkId": "5",
+      "type": "group",
+      "required": false,
+      "text": "Order",
+      "item": [
+        {
+          "required": false,
+          "linkId": "5.1",
+          "text": "Order: Indwelling catheters",
+          "type": "group",
+          "answerValueSet": "http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113762.1.4.1219.60",
+          "extension": [
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/cqf-expression",
+              "valueExpression": {
+                "language": "text/cql",
+                "expression": "\"HospitalBedsAndAccessoriesPrepopulation\".DeviceRequested"
+              }
+            },
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/questionnaire-itemControl",
+              "valueCodeableConcept": {
+                "coding": [
+                  {
+                    "system": "http://hl7.org/fhir/questionnaire-item-control",
+                    "code": "gtable"
+                  }
+                ]
+              }
+            }
+          ],
+          "item": [
+            {
+              "linkId": "5.1.1",
+              "text": "Item Description",
+              "type": "string"
+            },
+            {
+              "linkId": "5.1.2",
+              "text": "Usual Max/Mo.",
+              "type": "string"
+            },
+            {
+              "linkId": "5.1.3",
+              "text": "Quantity",
+              "type": "string"
+            },
+            {
+              "linkId": "5.1.4",
+              "text": "Refills",
+              "type": "string"
+            }
+          ],
+          "repeats": true,
+          "answerOption": []
+        },
+        {
+          "required": false,
+          "linkId": "5.2",
+          "text": "Order: Intermittent catheterization",
+          "type": "group",
+          "answerValueSet": "http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113762.1.4.1219.59",
+          "extension": [
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/cqf-expression",
+              "valueExpression": {
+                "language": "text/cql",
+                "expression": "\"HospitalBedsAndAccessoriesPrepopulation\".DeviceRequested"
```

Comment:
This should be moved down as the first item of the gtable and it should NOT be "DeviceRequested". It should be a drop down and populated from the valueset (the same as you did before).